napari/utils/notebook_display.py

```diff
@@ -0,0 +1,54 @@
+from io import BytesIO
+
+import imageio
```

Comment:
I'm trying to remember now, but I think in the rest of the codebase we've made our imports of imageio lazy, to make launch faster. We could put it inside `_repr_png_`. I'd probably also go `from imageio import imsave` too. Thoughts @jni?

Reply:
That's a good point, thanks. I've made those changes for you now.